src/bin/inx-chronicle/migrations/migrate_1_0_0_beta_31.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use chronicle::{
+    db::{collections::OutputCollection, MongoDb, MongoDbCollection, MongoDbCollectionExt},
+    types::stardust::block::output::{AliasId, NftId, OutputId},
+};
+use futures::TryStreamExt;
+use mongodb::{bson::doc, options::IndexOptions, IndexModel};
+use serde::Deserialize;
+
+pub const PREV_VERSION: &str = "1.0.0-beta.30";
```

Comment:
Why is this necessary? The version to run the migration from is already in the Cargo.toml manifest, right?

Reply:
Not if you've updated the application already.